packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -113,6 +117,13 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
+  /// Specifies whether a drag handle is shown when [enableDrag] is true.
+  ///
+  /// Defaults to [BottomSheetThemeData.hasDragHandle]. If that is also null,
+  /// behaves as true when [ThemeData.useMaterial3] is true, and otherwise
+  /// behaves as false.
```

Comment:
This would benefit from more documentation around color, size, and the relationship with `enableDrag`.

```suggestion
  /// Specifies whether a drag handle is shown when [enableDrag] is true.
  ///
  /// The drag handle appears at the top of the bottom sheet. The default color is XXX and can be customized using XXX. The default size is XXX and can be customized with XXX.
  ///
  /// Defaults to [BottomSheetThemeData.showDragHandle]. If that is also null,
  /// defaults to true when [ThemeData.useMaterial3] is true, and otherwise
  /// defaults to false.
```